configs/typescript/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@tsconfig/ember": "^3.0.0",
     "@tsconfig/node16": "^16.1.1",
+    "@tsconfig/node18": "^18.2.2",
```

Comment:
**TODO**

`@tsconfig/node16` will be removed once the packages that depend on this shared configuration drop Node 16 support.